uiplib/settings.py

```diff
@@ -10,17 +11,22 @@
 
 
 class ParseSettings:
+    """Argument Parser class."""
 
-    def __init__(self):
-        self.settings = self.get_settings_from_file()
-        self.settings.update(self.get_settings_from_cli())
+    def get_settings(self, args):
+        """Return the settings from parser."""
```

Comment:
so i am against this change. In the previous implementation, we just needed to initialize the `ParseSetting` Object once.
Here `get_settings`  will call the other two functions every time it is called. Which gives this class a huge overhead.